ReplacingDocker/Dockerfile.buildah

```diff
@@ -0,0 +1,6 @@
+FROM fedora
+
+RUN dnf -y install buildah; dnf -y clean all
+ENTRYPOINT ["/usr/bin/buildah"]
+WORKDIR /root
+ADD Dockerfile.fedora /root/Dockerfile
```

Comment:
Are you missing the Dockerfile.fedora file?

Reply:
I went ahead an put one together, things seemed to work fine after that.

Reply:
Yes removing this line.  Not needed.